templates/web/greenwich/header_logo.html

```diff
@@ -1,18 +1,35 @@
-    <a href="http://www.royalgreenwich.gov.uk/" id="site-logo">[% site_name %]</a>
+<a href="http://www.royalgreenwich.gov.uk/" id="site-logo">[% site_name %]</a>
+<ul id="greenwich-menu-desktop">
+    <li><a href="http://www.royalgreenwich.gov.uk/pay">Make a payment</a></li>
+    <li><a href="http://www.royalgreenwich.gov.uk/apply">Apply for it</a></li>
+    <li><a href="/">Report a problem</a></li>
+    <li><a href="http://www.royalgreenwich.gov.uk/contact">Contact us</a></li>
+</ul>
 </div>
-<div class="container main-menu-container">
-    <ul id="greenwich-menu-desktop">
+
+<div class="main-menu-container">
+    <div class="container page-header-container">
+        <!-- <ul id="greenwich-menu-desktop">
```

Comment:
Whoops, HTML comments left in here. Need to be removed.

Same for the commented-out `<h1>` in `templates/web/greenwich/around/intro.html`.